app/views/foreman_wreckingball/hosts/schedule_remediate.html.erb

```diff
@@ -0,0 +1,11 @@
+<% title _('Remediate %s') % @host %>
+
+<%= form_for ForemanTasks::Task.new, :url => submit_remediate_host_path(:id => @host.id, :status_id => @status), :html => {:id => 'schedule_remediate_form'} do |f| %>
+  <%= trigger_selector f, @triggering %>
```

Comment:
Can we remove `Set up recurring execution` here?

Reply:
The trigger_selector helper always sets up all possibles, probably going to do the selector generation myself instead